WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowSettingsViewController.swift

```diff
@@ -227,31 +252,67 @@ class BloggingRemindersFlowSettingsViewController: UIViewController {
     private func populateCalendarDays() {
         daysOuterStackView.addArrangedSubviews([daysTopInnerStackView, daysBottomInnerStackView])
 
-        let topRow = days[0..<4]    // First 4 days
-        let bottomRow = days[4..<days.count]    // Last 3 days
-        daysTopInnerStackView.addArrangedSubviews(topRow.map({ CalendarDayToggleButton(weekday: $0) }))
-        daysBottomInnerStackView.addArrangedSubviews(bottomRow.map({ CalendarDayToggleButton(weekday: $0) }))
-    }
-
-    /// Localized short weekday names, starting at the correct weekday for the current calendar
-    private let days: [String] = {
-        var calendar = Calendar.current
-        calendar.locale = Locale.autoupdatingCurrent
-        let firstWeekday = calendar.firstWeekday
-        var symbols = calendar.shortWeekdaySymbols
+        let createButton: (Int) -> CalendarDayToggleButton = { [unowned self] dayIndex in
+            let isSelected: Bool
+            let localizedDayIndex = (dayIndex + calendar.firstWeekday - 1) % calendar.shortWeekdaySymbols.count
+
+            let weekday = BloggingRemindersScheduler.Weekday(rawValue: localizedDayIndex) ?? {
+                // The rawValue above should always fall within 0 ..< 7 and this case
```

Comment:
can we be more concise with the comments? I don't think the history of a decision will be that useful to anyone who will work on this code in the future.

Reply:
This code was removed.

That said what's being documented is not the history of the decision, but the "why" of a very special scenario which may not be clear by just reading the code.  By documenting the "why" we make sure whoever reads the code can understand what is required to change it or remove it.